packages/subgraph/src/entities/Token.ts

```diff
@@ -1,10 +1,10 @@
 import { Address } from "@graphprotocol/graph-ts";
 
 import { Token } from "../../generated/schema";
-import { ERC20 } from "../../generated/templates/Token/ERC20"
+import { ERC20 } from "../../generated/LUSDToken/ERC20";
```

Comment:
Why this change? I used the one in template so this generic entity wouldn’t rely on an specific token code.

Reply:
LUSDToken and LQTYToken are concrete data sources in the manifest now. There's no longer a template.

Reply:
Right, I forgot to get back to this comment when I saw the changes below.